### PR TITLE
Fix link to GSN overview

### PR DESCRIPTION
**Pull request by nventuro**
@frangio is this the right way to include a link to a different component? IIRC documents that go through solidity-docgen sometimes require special treatment.

### CODE REVIEW

File: contracts/GSN/README.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@ _Available since v2.4.0._
 
 This set of contracts provide all the tools required to make a contract callable via the https://gsn.openzeppelin.com[Gas Station Network].
 
-TIP: If you're new to the GSN, head over to our xref:openzeppelin::gsn/what-is-the-gsn.adoc[overview of the system] and basic guide to xref:ROOT:gsn.adoc[creating a GSN-capable contract].
+TIP: If you're new to the GSN, head over to our xref:learn/sending-gasless-transactions.adoc[overview of the system] and basic guide to xref:ROOT:gsn.adoc[creating a GSN-capable contract].
```
Comment (frangio) [inline contracts/GSN/README.adoc]: solidity-docgen shouldn't make a difference here, but I believe this is the correct link: 

```suggestion
TIP: If you're new to the GSN, head over to our xref:learn::sending-gasless-transactions.adoc[overview of the system] and basic guide to xref:ROOT:gsn.adoc[creating a GSN-capable contract].
```

Comment (nventuro) [reply contracts/GSN/README.adoc]: I don't think it is, see this link here: https://github.com/OpenZeppelin/openzeppelin-contracts/blame/master/docs/modules/ROOT/pages/gsn.adoc#L5

I think the slash is only used when there are directories inside a component.

Comment (frangio) [reply contracts/GSN/README.adoc]: I don't understand. Don't we want to make a link to the `sending-gassless-transactions` article inside the `learn` component?

Comment (nventuro) [reply contracts/GSN/README.adoc]: Corect. See https://docs.antora.org/antora/2.2/asciidoc/page-to-page-xref/

Comment (frangio) [reply contracts/GSN/README.adoc]: Agreed offline that I'm right hehe.

**Pull request merged**